pontoon/base/static/css/style.css

```diff
@@ -948,7 +948,8 @@ header .right .select .menu {
   line-height: 22px;
 }
 
-#profile .menu li i.fa {
+#profile .menu li i.fa,
+#profile .menu li i.fab {
```

Comment:
You'll need to make a similar change here to properly position the GitHub link on the translate page:
https://github.com/mozilla/pontoon/blob/8f1663fd3ea53c9f29b3d2f9f90d92b56c62b5e2/frontend/src/core/user/components/UserMenu.css#L87

You'll need to add `.user-menu .menu li i.fab` to the rule.